apps/src/dom.js

```diff
@@ -37,14 +37,14 @@ exports.getTouchEventName = function(eventName) {
 };
 
 var addEvent = function(element, eventName, handler) {
-  element.addEventListener(eventName, handler, false);
-
   var touchEvent = exports.getTouchEventName(eventName);
   if (touchEvent) {
```

Comment:
yeah, I'd be careful here. I just modified this code a little recently so I could enable dragging of the resize bars (to change visualization column size for all apps or debug area size for applab only). Having it register both mousedown and touchdown may be important for my use cases.

One question which I've never fully understood is why do some of these buttons need this code path at all? Can't you just listen for "click"?